Changelog.rst

```diff
@@ -9,7 +9,33 @@ in the main branch & 5.3.x series, please see :ref:`whatsnew-5.3` for
 an overview of what's new in Celery 5.3.
 
 
-. _version-5.3.5:
+.. _version-5.3.6:
+
+5.3.6
+=====
+
+:release-date: 2023-11-22  7:15 P.M GMT+6
+:release-by: Asif Saif Uddin
+
```

Comment:
Maybe add a personal note mentioning the SQS fixes to differentiate this from the previous patch release that was focused on Python 3.12 @auvipy

Reply:
Good point